src/framework/Framework_Stability_001.test.js

```diff
@@ -0,0 +1,63 @@
+import {
+  pluginActivate,
+  pluginDeactivate,
+  restartFramework,
+  getControllerPluginData,
+} from '../commonMethods/commonFunctions'
+
+export default {
+  title: 'Framework Controller Robustness test',
+  description: 'Tests start/stop functionality of the controller module and repeats 100 times',
+  teardown: restartFramework,
```

Comment:
No need

Reply:
Updated as per the code review comment by removing the teardown method